wicket-auth-roles/pom.xml

```diff
@@ -39,6 +39,11 @@
 			<groupId>org.apache.wicket</groupId>
 			<artifactId>wicket-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.wicket</groupId>
+			<artifactId>wicket-core</artifactId>
+			<type>test-jar</type>
```

Comment:
Which classes are still in wicket-core:tests and are needed by the other wicket-xyz modules ?
IMO all such dependencies should be in `wicket-tester` module

Reply:
In this version, all test code remains in wicket-core:tests, including classes like WicketTester (with JUnit code) moved from /src/main to /src/test. Modules that require them must include wicket:core:test-jar. No wicket-tester is required (see other variant https://github.com/apache/wicket/pull/660).